apps/journeys-admin/src/components/TemplateCustomization/MultiStepForm/Screens/LinksScreen/LinksScreen.tsx

```diff
@@ -139,16 +142,57 @@ export function LinksScreen({
   return (
     <Stack
       alignItems="center"
-      sx={{ px: { xs: 2, md: 8 }, maxWidth: '1000px' }}
       gap={6}
+      sx={{
+        px: { xs: 2, sm: 18 },
+        width: '100%'
+      }}
     >
+      <Stack alignItems="center" sx={{ pb: 1 }}>
+        <Typography
+          component="h1"
+          variant="h4"
+          display={{ xs: 'none', sm: 'block' }}
+          gutterBottom
+          sx={{
+            mb: { xs: 0, sm: 2 }
+          }}
+        >
+          {t('Links')}
+        </Typography>
+        <Typography
+          component="h1"
+          variant="h6"
```

Comment:
```suggestion
          variant="h6"
```
one overrides the other, don't put component and variant together.